examples/all-clusters-app/all-clusters-common/gen/endpoint_config.h

```diff
@@ -391,232 +418,351 @@
+                ZAP_REPORT_DIRECTION(REPORTED), 0x0001, 0x0101, 0x0000, ZAP_CLUSTER_MASK(SERVER), 0x0000, { { 0, 65344, 0 } }      \
+            }, /* Reporting for cluster: "Door Lock", attribute: "lock state". side: server */                                     \
+            {                                                                                                                      \
+                ZAP_REPORT_DIRECTION(REPORTED), 0x0001, 0x0300, 0x0000, ZAP_CLUSTER_MASK(SERVER), 0x0000, { { 0, 65344, 0 } }      \
+            }, /* Reporting for cluster: "Color Control", attribute: "current hue". side: server */                                \
+            {                                                                                                                      \
+                ZAP_REPORT_DIRECTION(REPORTED), 0x0001, 0x0300, 0x0001, ZAP_CLUSTER_MASK(SERVER), 0x0000, { { 0, 65344, 0 } }      \
+            }, /* Reporting for cluster: "Color Control", attribute: "current saturation". side: server */                         \
+            {                                                                                                                      \
+                ZAP_REPORT_DIRECTION(REPORTED), 0x0001, 0x0300, 0x0003, ZAP_CLUSTER_MASK(SERVER), 0x0000, { { 0, 65344, 0 } }      \
+            }, /* Reporting for cluster: "Color Control", attribute: "current x". side: server */                                  \
+            {                                                                                                                      \
+                ZAP_REPORT_DIRECTION(REPORTED), 0x0001, 0x0300, 0x0004, ZAP_CLUSTER_MASK(SERVER), 0x0000, { { 0, 65344, 0 } }      \
+            }, /* Reporting for cluster: "Color Control", attribute: "current y". side: server */                                  \
+            {                                                                                                                      \
+                ZAP_REPORT_DIRECTION(REPORTED), 0x0001, 0x0300, 0x0007, ZAP_CLUSTER_MASK(SERVER), 0x0000, { { 0, 65344, 0 } }      \
+            }, /* Reporting for cluster: "Color Control", attribute: "color temperature". side: server */                          \
+            {                                                                                                                      \
+                ZAP_REPORT_DIRECTION(REPORTED), 0x0001, 0x0402, 0x0000, ZAP_CLUSTER_MASK(SERVER), 0x0000, { { 0, 65344, 0 } }      \
+            }, /* Reporting for cluster: "Temperature Measurement", attribute: "measured value". side: server */                   \
+            {                                                                                                                      \
+                ZAP_REPORT_DIRECTION(REPORTED), 0x0002, 0x0006, 0x0000, ZAP_CLUSTER_MASK(SERVER), 0x0000, { { 0, 65344, 0 } }      \
```

Comment:
It seems like all the `{ 1, 65534, 0}` values has been changed to `{ 0, 65534, 0}`. Do you know why ?

Reply:
minOrSource is set to 0 in the database.

Reply:
Ah yeah. I feel like I already asked that... Sorry.